test/export/test_export.py

```diff
@@ -5545,7 +5461,7 @@ def forward(self, x):
         unflattened = unflatten(ep)
         self.assertTrue(torch.allclose(unflattened(*inps), M2()(*inps)))
 
-    @testing.expectedFailureRetraceability  # Retracing tensor constants results in buffers
+    #@testing.expectedFailureRetraceability  # Retracing tensor constants results in buffers
```

Comment:
So this goes away, is that good or bad? Just want to make sure that if there's still something to fix, we keep an expected failure.

Reply:
The failure was actually not related to tensor constants lol. It was just some graph mismatch.